### ISSUE TITLE
browser.umd -- Cannot read properties of undefined (reading 'split') 

### ISSUE THREAD

**Issue by clathey123** [Automattic/mongoose#12639]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Mongoose version

6.7.0

### Node.js version

14.21.0

### MongoDB server version

4.11.0

### Typescript version (if applicable)

n/a

### Description

On Sunday I installed the "validator" node module and then ran an "audit fix". One of these two actions (likely the audit fix) caused my site to stop working. I am now getting the error noted in the subject line. I've done some reading on github and noted that others have had this same issue (see issue #11943). Has this issue been resolved or is it scheduled to be resolved in the near future?  If it has been resolved, what do I need to do to take advantage of the fix (I am relatively new to the world of website development so simple instructions would be greatly appreciated).

FYI... I am using reactjs with node. I am attaching the full text of the error that I am receiving when I try to launch my application. I am also adding a print screen of the information display in response to a request to display the version of mongoose that I am using.

![broswer umd error](https://user-images.githubusercontent.com/86580162/199375675-12e2f7f7-d6a5-4201-add9-12495a25f056.jpg)
![Mongoose Version](https://user-images.githubusercontent.com/86580162/199375691-53088088-e4df-4578-8724-a800026a589c.jpg)

Please let me know if there is any other information that I can provide.

Thank you for the assist!!!   Cheryl

### Steps to Reproduce

This bug occurs when I run my project using "npm run start:dev" in the CMD prompt.  The error is trapped (see previous) and displayed before the application opens.  Since this has been reported previously, I am assuming that I do not need to provide anymore specifics.... Please let me know if I am not correct about this assumption.

### Expected Behavior

The application should load without the error noted above.

**Comment by vkarpov15** [comment]
Without a more complete repro script, there's no way we'll be able to figure out what's going on here. The stack trace points to minified code, so it isn't helpful. All I can really tell is that you're running Mongoose in the browser, which is probably not what you're trying to do based on your description.

**Comment by clathey123** [comment]
Hello!  Thank you for your comment.  The error was happening when I launched my site, by the time I was able to 'inspect' the error had already occurred, so it would be very difficult to provide a repro script.  I ended up reverting to a prior version of mongoose (6.5) and then my site was able to launch and it works as expected.  I am hoping that someone else stumbles on this issue and will have an easier time providing you with a repro script and non-minified code but...  as of now, I am considering this item closed.  Have a great day!   Cheryl
